### PR TITLE
refs #13528 - docker images are changing to manifests

### PR DESCRIPTION
**Pull request by komidore64**

### CODE REVIEW

File: lib/hammer_cli_foreman_docker/docker_manifest.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ def extend_data(record)
       class InfoCommand < HammerCLIKatello::InfoCommand
         output do
           field :id, _("ID")
-          field :image_id, _("Image ID")
+          field :manifest_id, _("Manifest ID")
           field :size, _("Size")
```
Comment (bbuckingham) [inline lib/hammer_cli_foreman_docker/docker_manifest.rb]: The information provided for a Docker manifest is different than what was previously provided for an image; therefore, we need to update the output.

Refer to : https://github.com/Katello/katello/blob/master/app/views/katello/api/v2/docker_manifests/show.json.rabl for details on what is currently available for a manifest.

File: lib/hammer_cli_foreman_docker/docker_manifest.rb [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
+      command_name 'manifest'
+      desc _('Manage docker manifests')
 
       class ListCommand < HammerCLIKatello::ListCommand
         include ::HammerCLIKatello::LifecycleEnvironmentNameResolvable
         output do
           field :id, _("ID")
-          field :image_id, _("Image ID")
+          field :manifest_id, _("Manifest ID")
           field :size, _("Size")
```
Comment (bbuckingham) [inline lib/hammer_cli_foreman_docker/docker_manifest.rb]: Similar to the comment below, the output for the list command needs to be adapted to the information returned for a manifest.

Eg. size is not available, a manifest can only have 1 tag, that tag has a name...etc.

**Comment by bbuckingham** [comment]
The information available for tags is different from what was previously supported with Docker v1; therefore, we need to update the commands accordingly.  

Refer to: https://github.com/Katello/katello/blob/master/app/views/katello/api/v2/docker_tags/show.json.rabl

**Comment by komidore64** [comment]
@bbuckingham updated.

**Comment by bbuckingham** [comment]
docker_tag.rb should be updated.  We should now reference the manifest vs image.

**Comment by komidore64** [comment]
updated

**Comment by bbuckingham** [comment]
ACK

**Pull request merged**